eahub/profiles/models.py

```diff
@@ -114,7 +120,7 @@ def visible_to_user(self, user):
         if user.is_superuser:
             return self.all()
         return self.filter(
-            models.Q(is_public=True, is_approved=True) | models.Q(user_id=user.pk)
+            models.Q(is_publicly_visible=True, is_approved=True) | models.Q(user_id=user.pk)
```

Comment:
```is_publicly_visible``` is set to ```visibility=Visibility.PRIVATE or visibility=Visibility.INTERNAL``` so means the same as the exclude code. 

But I don't care too much

Reply:
I wouldn't care about it either, but loading objects in RAM instead of a single SQL query is usually hard for django and doesn't go unnoticed. I'll create a ticket then.